project/Versions.scala

```diff
@@ -26,7 +26,9 @@ object Versions {
 
   val Akka            = "2.3.4"
   val Cassandra       = "3.11.3"
-  val CassandraDriver = "3.7.2"
+  // val Cassandra       = "3.6"
+  val CassandraDriver = "3.2.0-yb-10"
+>>>>>>> 5babbcaf... Use partition-aware policy for Spark load balancing and partitioning
```

Comment:
unfinished-merge artefact? (It's cleaned up in a follow-up commit, but better to fix it here to keep a clean history.